lib/puma/control_cli.rb

```diff
@@ -220,7 +220,7 @@ def send_signal
     end
 
     def run
-      start if @command == "start"
+      return start if @command == "start"
```

Comment:
Prior to this change, `send_request` was being called with `@command == "start"`. The control server does not respond to this command, so returns a 404, which then ultimately calls `exit 1`, making it very difficult to figure out why my test was failing silently. This early return short-circuits all that mess.